src/Microsoft.Crank.AzureDevOpsWorker/DevopsMessage.cs

```diff
@@ -38,13 +39,15 @@ public DevopsMessage(ServiceBusReceivedMessage message)
             TaskInstanceId = (string)message.ApplicationProperties["TaskInstanceId"];
             AuthToken = (string)message.ApplicationProperties["AuthToken"];
 
-            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic",
-                    Convert.ToBase64String(Encoding.ASCII.GetBytes(":" + AuthToken)));
+            _httpClient.DefaultRequestHeaders.Authorization =
+                new AuthenticationHeaderValue("Basic",
```

Comment:
You can target type here as well no?

Reply:
Yes, I believe so. I'll have to rebase this PR and address the merge conflicts, not sure how I missed the reply here. 😬